microcloud/test/suites/preseed.sh

```diff
@@ -83,4 +83,47 @@ EOF
   validate_system_lxd micro04 4 disk1 1 enp6s0 10.1.123.1/24 10.1.123.100-10.1.123.254 fd42:1:1234:1234::1/64
   validate_system_microceph micro04 disk2
   validate_system_microovn micro04
+
+  reset_systems 3 3 2
```

Comment:
Thanks, yeah for most of these I've put a comment or echo that summarizes the run right before it, I'll add some here too. 

As for checking the result of the cluster, that's what those `validate` helpers do. Since there's only so many resulting configurations after clustering, I just made some helpers that check for the correct config/devices based on the expected input arguments.

Reply:
https://github.com/canonical/microcloud/blob/10dc1dc8cbd77d5bf993403ea9b27970465c248d/microcloud/test/includes/microcloud.sh#L258C5-L258C5

Reply:
Ah OK thanks. Comments are good thanks!